packages/web-components/@sfx/sayt/src/sayt.ts

```diff
@@ -64,15 +94,103 @@ export default class Sayt extends LitElement {
     this.visible = true;
   }
 
+  /**
+   * Makes SAYT visible if the event refers to the correct SAYT component.
+   *
+   * @param event An event that can contain a searchbar ID.
+   */
+  showCorrectSayt(event: CustomEvent) {
+    if (this.isCorrectSayt(event)) {
+      this.showSayt();
+    }
+  }
+
   /**
    * Changes the `visible` property to `false`.
    */
   hideSayt() {
     this.visible = false;
   }
 
+  /**
+   * Hides SAYT if the event refers to the correct SAYT component.
+   *
+   * @param event An event that can contain a searchbar ID.
+   */
+  hideCorrectSayt(event: CustomEvent) {
+    if (this.isCorrectSayt(event)) {
+      this.hideSayt();
+    }
+  }
+
+  /**
+   * Determines whether an event refers to the correct SAYT. This is true if
+   * a matching `searchbar` ID is specified, if the event has no `searchbar`
+   * ID specified, or if SAYT has no `searchbar` ID specified.
+   *
+   * @param event An event that can contain a searchbar ID for comparison.
+   */
+  isCorrectSayt(event: CustomEvent) {
```

Comment:
Should the return value type be indicated ('boolean'), or is it unnecessary?

Reply:
It should be there! Good catch.